### PR TITLE
Third homework about

### PR DESCRIPTION
**Pull request by ryvik13**
only about page for now

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @ryvik13, please add all files from the task to the pull request. I'm changing it to draft, undraft it, when everything is ready to review.
image

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.html:1]: Please change the name of the file back to `index.html` - it is the main starting point for any web-site

Comment (vladyslav-yermolin-moc) [reply homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.html:1]: And, BTW, all these files should be placed in GamesProject, not in this
image

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css [hunk 2]
```diff
@@ -0,0 +1,232 @@
+/*header*/
+
+.header {
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css:3]: Please make the page appearance closer to design view:
- logo top margin
- align of the h1
- font of the text
image

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.css [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
+}
+
+.icon-inst img {
+  background-color: #FFF;
+  height: 20px;
+  width: 20.000892639160156px;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.css:265]: Wow, what an accuracy))))
And persistence - triple kill)))
Please correct: we are using integer px sizes and one time of width and height is enough.

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.css [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+  width: 20.000892639160156px;
+  height: 20px;
+  left: 161px;
+  top: 11px;
+  width: 20.000892639160156px;
+  height: 20px;
+  position: absolute;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.css:272]: I think you can insert an image inside of the link without absolute. Just add proper size

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.html [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="games-contain">
+                <div class="games-content">
+                    <h1 class="text-heading">
+                        Study project "Games"
+                    </h1>
+                    <div class="games-cards">
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.html:39]: `article` maybe?

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.css (hunk 3 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.css]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css (hunk 2 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css:3]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.html (hunk 1 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.html]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.html (hunk 5 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.html:39]: I think the article is more general

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css [hunk 6]
```diff
@@ -0,0 +1,234 @@
+/*header*/
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css:1]: Please move 
- css files to `GamesProject/styles` folder
- html files to `GamesProject` folder

Delete folder `2-css-ryvik13`cause all your work should be placed in the `GamesProject` folder

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
+}
+
+.icon-inst img {
+  background-color: #FFF;
+  height: 20px;
+  width: 20.000892639160156px;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css:217]: fix these numbers after `.` here and everywhere in your files

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+  width: 20.000892639160156px;
+  height: 20px;
+  left: 161px;
+  top: 11px;
+  width: 20.000892639160156px;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css:222]: you still have 3 `width` and 3 `height`

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css (hunk 6 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css:1]: fixed

File: homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css (hunk 7 above)
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css:217]: fixed but its looks strange

**Pull request merged**